packages/protocol/contracts/layer2/based/anchor/PacayaAnchor.sol

```diff
@@ -24,6 +24,7 @@ abstract contract PacayaAnchor is OntakeAnchor, ITaiko {
 
     /// @notice Golden touch address is the only address that can do the anchor transaction.
     address public constant GOLDEN_TOUCH_ADDRESS = 0x0000777735367b36bC9B61C50022d9D0700dB4Ec;
+    uint256 public constant BASEFEE_MIN_VALUE = 100000000000; // 0.1 gwei
```

Comment:
Isn't this `100 gwei`? Or am I missing something here.
![image](https://github.com/user-attachments/assets/ac57375d-4c21-45ed-8e74-ded45a1fc5c1)